requirements.txt

```diff
@@ -1,10 +1,9 @@
-molo.core==3.12.3
-wagtailmedia==0.1.1
+molo.core==4.0.0
```

Comment:
@nathanbegbie please check if there was anything added from 3.12.3 to 4.0.0 in molo core that we need to add to free basics. same goes for the plugins